packages/crypto-wallet-core/src/transactions/erc20/index.ts

```diff
@@ -40,13 +40,13 @@ export class ERC20TxProvider extends ETHTxProvider {
       const amounts = [];
       for (let recipient of recipients) {
         addresses.push(recipient.address);
-        amounts.push(toBN(recipient.amount));
+        amounts.push(toBN(BigInt(recipient.amount).toString()));
       }
       const multisendContract = this.getMultiSendContract(contractAddress);
       return multisendContract.methods.sendErc20(tokenAddress, addresses, amounts).encodeABI();
     } else {
       const [{ address, amount }] = params.recipients;
-      const amountBN = toBN(amount);
+      const amountBN = toBN(BigInt(amount).toString());
```

Comment:
getting the following error here

`Error: [number-to-bn] while converting number 1.15207373271889e+26 to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported. Given value: "1.15207373271889e+26"`

weird considering the use of the similar changes in `checkValidTxAmount` no longer throw an error. assuming this error will occur on the same `index.ts` changes for polygon and native eth